hugolib/resource_chain_test.go

```diff
@@ -699,7 +699,7 @@ Min HTML Remote: {{ ( resources.GetRemote "%[1]s/mydata/html1.html" | resources.
 		}, func(b *sitesBuilder) {
 			b.AssertFileContent("public/index.html", `Min CSS: h1{font-style:bold}`)
 			b.AssertFileContent("public/index.html", `Min CSS Remote: h1{font-style:bold}`)
-			b.AssertFileContent("public/index.html", `Min JS: var x;x=5,document.getElementById(&#34;demo&#34;).innerHTML=x*10`)
+			b.AssertFileContent("public/index.html", `Min JS: var x=5;document.getElementById(&#34;demo&#34;).innerHTML=x*10`)
 			b.AssertFileContent("public/index.html", `Min JS Remote: var x;x=5,document.getElementById(&#34;demo&#34;).innerHTML=x*10`)
```

Comment:
Not that there was ever an ambition that these tests should somehow "unit test" the upstream library, I cannot help questioning the behaviour above: Why is is behaving differently when applied to the same JS file when pulled in remotely?

Reply:
> Why is it behaving differently when applied to the same JS file when pulled in remotely?

Because it's not the same JS file.

The "Min JS" test uses this data:
<https://github.com/gohugoio/hugo/blob/master/hugolib/resource_chain_test.go#L942-L944>

The "Min JS Remote" test uses this data:
<https://github.com/gohugoio/hugo/blob/master/hugolib/resource_chain_test.go#L568>

Reply:
I've asked a question about the differences:
<https://github.com/tdewolff/minify/issues/455>

Reply:
Differences unexpected, resolved with new release 2.9.29.